camus-sweeper/pom.xml

```diff
@@ -17,6 +17,10 @@
     </properties>
 
     <dependencies>
+      <dependency>
+        <groupId>com.linkedin.camus</groupId>
+        <artifactId>camus-schema-registry</artifactId>
```

Comment:
To be able to reference `com.vinted.camus.schemaregistry.ConnectivityCheck` in `camus-sweeper`.